### ISSUE TITLE
Firebase auth: Operation timeout issues on login, using social media login eg. google

### ISSUE THREAD

**Issue by vaibhav-pande** [firebase/firebaseui-web#791]
### [REQUIRED] Describe your environment
  * Operating System version: Windows 10
  * Browser version: Chrome 
  * Firebase SDK version: 7.19.1
  * Firebase Product: auth
 
 ### [REQUIRED] Describe the problem
We are using firebase on webapp which acts as an SSO provider for our connected applications, deployed on appengine.
At times, after we enter the credentials for social media login eg. google, it doesn't move ahead and gets stuck, **giving operation time out error.**
When we inspected on console, we could see the error with iframewrapper.js which is part of the highlighted javascript.(iframewrapper.js)
**Even though having good network bandwidth, many users are facing this issue they are able to login only after multiple retries.**
**What is the probable cause for this issue and solution?
Is this issue resolved post 7.19.1 Firebase release? We have continuously updated the versions from 7.13.2 till 7.19.1.**
We could see many issues logged on various forums regarding operation time out, but we couldn't get satisfactory resolution till date.

#### Steps to reproduce:
1) Go to sso sign in page
2) Click on social media login option eg. google
3) Enter credentials
4) It comes back to sso sign in page giving operation time out error on google icon

Attaching screenshot for reference:
Screenshot 1:
![image](https://user-images.githubusercontent.com/46163092/98247827-215b7a80-1f9a-11eb-9f14-51467b9e436d.png)

Screenshot2:
![image](https://user-images.githubusercontent.com/46163092/98247897-3a642b80-1f9a-11eb-8a6d-6f29d83c4749.png)

**Comment by vaibhav-pande** [comment]
Few more screenshot for reference:
![image](https://user-images.githubusercontent.com/46163092/98804421-2dcd5080-243c-11eb-8c4b-57c61b5ebf82.png)

![image](https://user-images.githubusercontent.com/46163092/98804444-36be2200-243c-11eb-8806-65ed17088103.png)

**Comment by bojeil-google** [comment]
Hey @vaibhav-pande, please provide a link to your app or a reliable way to reproduce this issue. If sensitive info is involved, please file a ticket with the [support](https://firebase.google.com/support) team.

Given there are no clues provided, I can only speculate that you may be using a service worker and caching offline content which is messing up with the path `/__/auth`. You have to exclude that path. I have seen a similar error when developers do this.

**Comment by thespacedeck** [comment]
Hi there,

Having the same issue with custom Identity Provider. Was this resolved? Any idea how to resolve this?

Let me know :)

**Comment by mogwely** [comment]
Having the same issue. This is not the first network error I experience with firebase auth.
I searched everything online and all I see is very basic troubleshooting advice like make sure your internet is connected or irrelevant stuff like change <form> to <div>. Please someone from firebase get to the bottom of this.

**Comment by jhades** [comment]
I'm getting similar reports as well.

**Comment by dackers86** [comment]
Thank you for taking the time to submit this issue.

A new release is planned shortly, which may resolve this request. 

In the meantime, we will close this issue, if this is not resolved in the new version - please feel free to create a new issue.
